### ISSUE TITLE
Step up git revision

### ISSUE THREAD

**Issue by klslz** [xiph/flac#84]
When downloading sources from flac git,  it still shows version 1.3.2  - binary,library,...

Due to rather limited number of releases,  people use the git sources as base. 
And that's causing confusion.

Beside that you also offer 1.3.2 from 1.1.2017  as stable on https://ftp.osuosl.org/pub/xiph/releases/flac/

Of course those 2 - git and stable - have nothing in common.

Files generated with the git version will also get below vendor string written into the Vorbis header:
"vendor string: reference libFLAC 1.3.2 20170101"

Which is IMO not OK. It wouldn't reflect all the changes that went into flac-git recently. 

I'd think stepping up the revision in git or introducing a development branch would make a lot of sense to me and live much easier. Thx a lot.

**Comment by erikd** [comment]
First of all, I don't think anyone should be building FLAC from git *unless* they build from the latest git tag.

It would be possible to add a git revision to the version number but changing the vendor string on a regular basis would be a huge pain in the neck because every time that string changes, changes are required in the tests.

**Comment by soundchekk** [comment]
I agree that using git sources bears a slight risk.

However. The main issue is: Anyone don't want to wait two years for an update, especially if some interesting features and corrections were introduced in between.

With a little greping and seding I got my git based version stepped up to 1.3.3-git (including the Copyright years ) within 15 minutes of research. The next one will take me just two minutes.


#### step up revision and dates
find ./ -type f -exec sed -i 's/ 2017 / 2018 /g' {} \;
find ./ -type f -exec sed -i 's/20170101/20182911/g' {} \;
find ./ -type f -exec sed -i 's/1_3_2/1_3_3-git/g' {} \;
find ./ -type f -exec sed -i 's/1\.3\.2/1\.3\.3-git/g' {} \;
find ./ -type f -exec sed -i 's/2011-2016/2011-2018/g' {} \;
###

First regression tests look great.

**Comment by soundchekk** [comment]
I did realize that the vendor string gets used differently of apps using libFLAC.
I also realized that you add the libFLAC release date to the data file by default. And that I thought is actually a good idea. 
It allows me to somehow identify the "encoding" algorithm or whatever flac binary feature that went into the data file at the time of it's generation. I do realize that's twisting the actual meaning of that field.

Since you  put "20170101" - as provided by the current git sources - into it anyway. I thought
updating it accordingly just makes sense.
